.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -19,12 +19,12 @@ pr:
     - vhdbuilder/packer/*windows*
     - vhdbuilder/packer/**/*windows*
 
-pool:
-  name: $(POOL_NAME)
-
 variables:
 - name: CONTAINER_IMAGE
-  value: 'mcr.microsoft.com/oss/azcu/go-dev:v1.38.1'
```

Comment:
why the downgrade here?

Reply:
Ubuntu images will not complete the convert-to-vhd-blob step without downgrading to 1.38.0. I have not been able to figure out why.